src/XrdCrypto/XrdCryptosslX509.cc

```diff
@@ -507,8 +507,8 @@ const char *XrdCryptosslX509::IssuerHash(int alg)
       if (issueroldhash.length() <= 0) {
          // Make sure we have a certificate
          if (cert) {
-            char chash[15] = {0};
-            snprintf(chash,15,"%08lx.0",X509_NAME_hash_old(cert->cert_info->issuer));
+            char chash[30] = {0};
+            snprintf(chash, sizeof(chash), "%08lx.0", X509_NAME_hash_old(cert->cert_info->issuer));
```

Comment:
Yup, that's the correct format.  I believe it's to allow for collisions (so you can have a CA with `.0`, `.1`, `.2`, ....